pkg/cmd/admin/top/printer.go

```diff
@@ -0,0 +1,84 @@
+package top
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"strings"
+	"text/tabwriter"
+)
+
+type Info interface {
+	PrintLine(out io.Writer)
+}
+
+func Print(out io.Writer, headers []string, infos []Info) {
+	s := tabbedString(func(out *tabwriter.Writer) {
+		printHeader(out, headers)
+		for _, info := range infos {
+			info.PrintLine(out)
+			fmt.Fprintf(out, "\n")
+		}
+	})
+	fmt.Fprintf(out, "%s", s)
+}
+
+func printHeader(out io.Writer, columns []string) {
+	for _, col := range columns {
+		printValue(out, col)
+	}
+	fmt.Fprintf(out, "\n")
+}
+
+func printArray(out io.Writer, values []string) {
+	printValue(out, strings.Join(values, ","))
+}
+
+func printValue(out io.Writer, value interface{}) {
+	fmt.Fprintf(out, "%v\t", value)
+}
+
+func printBool(out io.Writer, value bool) {
+	if value {
+		printValue(out, "yes")
+	} else {
+		printValue(out, "no")
+	}
+}
+
+type scale struct {
+	scale uint64
+	unit  string
+}
+
+var (
+	mega = scale{20, "MiB"}
+	giga = scale{30, "GiB"}
+)
+
+func printSize(out io.Writer, size int64) {
```

Comment:
Where? You might be thinking about some other PR I was doing, that never merged in.